core/src/main/java/org/apache/iceberg/rest/CatalogHandlers.java

```diff
@@ -108,6 +109,14 @@ public CommitFailedException wrapped() {
     }
   }
 
+  private static <T> List<T> paginate(List<T> result, int pageStart, int pageSize) {
+    if (pageStart >= result.size()) {
+      return Collections.emptyList();
+    }
```

Comment:
Style Nit: Newline after if

Reply:
would be nice if this also showed in the PR by adding a linter ;-)